log/iface.go

```diff
@@ -0,0 +1,17 @@
+package log
+
+var _ Logger = logger{}
+
+// Logger is a convenience interface
+type Logger interface {
```

Comment:
I typically do not like packages defining and exposing exported interfaces. 

Go interfaces let you invert the dependency (as it should be), such that the caller gets to define what it wants. 

However, a `log` package is an exception to the above for the convenience of callers asking for a `log.Logger` vs. forcing them to define an interface each time.